src/katex-header.html

```diff
@@ -0,0 +1,18 @@
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.10.0/dist/katex.min.css"
```

Comment:
Is it common to put a file like this in `src/`? I think ideally we want only rust files here.